### DISCUSSION TITLE
Guess country

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#723] [Galaxy Trivia] [answered]
🟦🟨🔱🟦
🟩✨🌙🟩
🟦🟥⬜🕸️

**Comment by peterbaccaro** [comment]
1. BARBADOS
2. TURKMENISTAN
3. UNITED KINGDOM

![flags3](https://github.com/ParanoidUser/yolo/assets/74666079/96df64a6-d59e-40b0-aa5c-b1087c5ce14d)

**Reply by ParanoidUser** [reply]
Is something missing from the Turkmenistan flag?
![image](https://github.com/ParanoidUser/yolo/assets/5120290/71fb3004-bff9-4b12-ba29-3424226170f9)

**Reply by peterbaccaro** [reply]
It's a simplified version of the flag without the carpet...

https://en.wikipedia.org/wiki/List_of_Turkmen_flags

**Comment by thuongtruong1009** [comment, marked as answer]
Based on the provided emojis, it appears to represent the flag of Greece, which is 🇬🇷.
